frontend/src/store/episode/actions.ts

```diff
@@ -10,4 +10,12 @@ const loadEpisode = createAsyncThunk<Episode, number, AsyncThunkConfig>
   return episode;
 });
 
-export { loadEpisode };
+const createEpisode = createAsyncThunk<Episode, EpisodeCreatePayload, AsyncThunkConfig>
```

Comment:
you need to create your own reducer, actions and etc called `configurateEpisode`